### ISSUE TITLE
Add ZedX Camera to Isaac

### ISSUE THREAD

**Issue by andrewjong** [castacks/AirStack#104]

**Comment by andrewjong** [comment]
@OasisArtisan would you like to help with this one?

**Comment by andrewjong** [comment]
@bsj970 will help with this

**Comment by andrewjong** [comment]
blocked by https://github.com/stereolabs/zed-isaac-sim/issues/10

**Comment by andrewjong** [comment]
This is now unblocked! Official extension now supports Isaac 4.2.0

**Comment by andrewjong** [comment]
Looks like we need a CUDA docker image?

**Comment by andrewjong** [comment]
blocked again: https://github.com/stereolabs/zed-ros2-wrapper/issues/316

**Comment by andrewjong** [comment]
I found out a couple of issues
1. the ZedX isaac sim version only has the 2.2mm focal length. However we've purchased the 4mm, so the focal length doesn't match.
2. the resolution of the ZedX in isaac sim is weirdly 1080p, but the camera output is 540x960. It seems they downsample by x2 for sim efficiency. this is okay, but it's weird. also weird is that 1080p doesn't even match the actual sensor resolution, which is really 1200x1920. The downsampled resolution should be 600x960.
3. The ZedX omnigraph node isn't setup well for multi-robot namespacing. it also requires the zed-ros2-wrapper to run within the isaac container, it can't be in a different robot container
4. we don't really need what the rest of the ZedX API provides, which is the IMU and mapping.

**Comment by andrewjong** [comment]
low priority, we can just use isaac sim cameras in stereo
